core/resources/shaders/point.vs

```diff
@@ -46,10 +53,18 @@ void main() {
         float st = sin(a_rotation);
         float ct = cos(a_rotation);
 
+        vec2 vertexPos = a_position;
+        float de = a_extrude.y;
+
+        if (de != 0.0) {
+            float dz = u_map_position.z - abs(u_tile_origin.z);
+            vertexPos.xy += dz * unitExtrude[int(a_extrude.x)].xy * de;
```

Comment:
Accessing an array element with a non-constant or non-loop variable makes some drivers not receive this statement. Will follow-up on a fix related to this.